truenas_installer/disks.py

```diff
@@ -0,0 +1,69 @@
+from dataclasses import dataclass
+import json
+import re
+
+from .utils import GiB, run
+
+__all__ = ["list_disks"]
+
+# Minimum required space for an installation.
+# Docs state 8 GiB is the bare minimum, but we specify 8 GB here for wiggle room.
```

Comment:
You say you specify 8GB, bit its 8GiB below

Comment or code is wrong?

Reply:
This will only be important for people installing TN on 8GB USB drives, I suspect nobody does that :) This number never appears anywhere in the UI, it's just the threshold to filter out obviously bogus disks.

Reply:
Well, dont forget VMs

Reply:
You are right, fixed